app/src/js/components/Sidebar/sidebar.js

```diff
@@ -59,10 +64,14 @@ const Sidebar = ({
                 {d[0] && <li>
                   <Link
                     className={classes}
-                    to={(routeLocation) => ({
-                      pathname: path,
-                      search: locationQueryParams.search[path] || getPersistentQueryParams(routeLocation),
-                    })}
+                    to={location}
+                    onClick={(e) => {
```

Comment:
Does this fix an issue? Was there a conflict here where e.g. clicking the link was doing something unexpected that needed to be prevented?